### PR TITLE
Do not poll counters in bulk mode during initialization for objects that support bulk per CLI option

### PR DESCRIPTION
**Pull request by stephenxs**
Do not poll counters in bulk mode during initialization for objects that support bulk per CLI option
1. Read the counter groups that support bulk mode from `CONFIG_DB.DEVICE_METADATA|localhost.supporting_bulk_counter_groups`
2. Generate CLI options to syncd
3. Based on CLI options syncd records the counter groups that support bulk mode. For those objects, we do not need to poll them in bulk mode during initialization

### CODE REVIEW

File: syncd/FlexCounter.cpp [hunk 1]
```diff
@@ -1135,37 +1137,32 @@ void FlexCounter::addCounterPlugin(
...
+                {FLOW_COUNTER_PLUGIN_FIELD, COUNTER_TYPE_FLOW}};
+
+            auto counterTypeRef = plugIn2CounterType.find(field);
+            if (counterTypeRef != plugIn2CounterType.end())
+            {
+                getCounterContext(counterTypeRef->second)->addPlugins(shaStrings);
+
+                if (m_noDoubleCheckBulkCapability)
+                {
+                    getCounterContext(counterTypeRef->second)->setNoDoubleCheckBulkCapability(true);
```
Comment (DavidZagury) [inline syncd/FlexCounter.cpp:1158]: @stephenxs the function setNoDoubleCheckBulkCapability does not exists

**Review by DavidZagury — CHANGES_REQUESTED** [review]
Fix missing function

**Comment by stephenxs** [comment]
ci 4481 + rerun 587 passed

**Review by keboliu — APPROVED** [review]

**Comment by stephenxs** [comment]
ci 4519 passed

**Comment by stephenxs** [comment]
Force push to squash and avoid conflict.

**Pull request closed without merging**